packages/okta-angular/src/okta/models/user-claims.ts

```diff
@@ -0,0 +1,29 @@
+/**
+ * @export
+ * @interface UserClaims
+ *
+ * This interface represents the union of possible known claims that are in an
+ * ID Token or returned from the /userinfo response and depend on the
+ * response_type and scope parameters in the authorize request
+ */
+export interface UserClaims {
+  auth_time?: Number;
+  aud?: string;
+  email?: string;
+  email_verified?: Boolean;
+  exp?: Number;
+  family_name?: string;
+  given_name?: string;
+  iat?: Number;
+  iss?: string;
+  jti?: string;
+  locale?: string;
+  name?: string;
+  nonce?: string;
+  preferred_username?: string;
+  sub: string;
+  updated_at: Number;
+  ver: Number;
+  zoneinfo: string;
```

Comment:
`updated_at`, `ver`, and `zoneinfo` may not be included in the `/userinfo` response, so we should make these optional. See the [Standard Claims](http://openid.net/specs/openid-connect-core-1_0.html#StandardClaims).